macro-fullcalendar-ui/src/main/resources/Calendar/Macro.xml

```diff
@@ -262,19 +263,7 @@
 /************************************
  * Configure RequireJS to use jQuery-UI
  ************************************/
-require.config({
-  paths: {
-    // Use the 'r=1' query string in order to inform the require that there's no need to append .js at the end of the path.
-    'jquery-ui': "$xwiki.getURL('Calendar.JQueryUI', 'jsx', 'defer=false&amp;minify=false')",
-    'moment': "$xwiki.getDocument('Calendar.Moment').getAttachmentURL('moment-2.8.4.min.js', 'download', 'r=1')",
-    'fullcalendar': "$xwiki.getURL('Calendar.FullCalendar', 'jsx', 'defer=false&amp;minify=false')",
-    'gcal': "$xwiki.getDocument('Calendar.FullCalendar').getAttachmentURL('gcal.min.js', 'download', 'r=1')"
-  },
-  shim: {
-    'gcal': ['fullcalendar']  
-  }
-});
-require(['jquery', 'jquery-ui', 'fullcalendar', 'gcal'], function ($) {
+require(['jquery', 'jqueryui', 'fullcalendar', 'gcal'], function ($) {
```

Comment:
Where are the 'fullcalendar' and 'gcal' modules defined? I see you removed the require configuration. I was expecting to see the paths changed from JSX to WebJar paths.

Reply:
They are defined in included page Calendar.FullCalendar
https://github.com/xwiki-contrib/macro-fullcalendar/pull/18/files#diff-92d955c79230973a51bbabf65ebaebb8R128